src/services/item/plugins/app/chatBot/service.ts

```diff
@@ -0,0 +1,84 @@
+import OpenAI from 'openai';
+
+import { ChatBotMessage, PermissionLevel } from '@graasp/sdk';
+
+import { OPENAI_GPT_VERSION } from '../../../../../utils/config';
+import {
+  MemberCannotWriteItem,
+  OpenAIBadVersion,
+  OpenAIBaseError,
+  OpenAILengthError,
+  OpenAIQuotaError,
+  OpenAITimeOutError,
+} from '../../../../../utils/errors';
+import { Repositories } from '../../../../../utils/repositories';
+import { validatePermission } from '../../../../authorization';
+import { GPTVersion } from './interfaces/gptVersion';
+
+export class ChatBotService {
+  async post(
+    actorId: string | undefined,
+    repositories: Repositories,
+    itemId: string,
+    body: Array<ChatBotMessage>,
+    gptVersion?: GPTVersion,
+  ) {
+    const { memberRepository, itemRepository } = repositories;
+
+    // check member exists
+    if (!actorId) {
+      throw new MemberCannotWriteItem();
+    }
+    const member = await memberRepository.get(actorId);
+
+    // check item exists? let post fail?
```

Comment:
Yes fail if the item does not exist, as we do not want for people to use ghost items.